test/models/test_orcid_token.py

```diff
@@ -0,0 +1,26 @@
+from profiles.models import OrcidToken
+from profiles.utilities import expires_at
+
+
+def test_it_can_be_printed():
+    orcid_token = OrcidToken('0000-0002-1825-0097', '1/fFAGRNJru1FTz70BzhT3Zg', expires_at(1234))
+
+    assert repr(orcid_token) == "<OrcidToken '0000-0002-1825-0097'>"
```

Comment:
I assume it's the safest option to avoid printing the token value implicitly, in case it gets logged or dumped somewhere public

Reply:
That's the ORCID rather than the access token.

Reply:
(Could add 'for'.)

Reply:
yes, that's why this way is safe :+1: